source/server/server.h

```diff
@@ -63,7 +63,7 @@ namespace Server {
   GAUGE(total_connections, Accumulate)                                                             \
   GAUGE(uptime, Accumulate)                                                                        \
   GAUGE(version, NeverImport)                                                                      \
-  HISTOGRAM(initialization_time_ms)
+  HISTOGRAM(initialization_time, Milliseconds)
```

Comment:
again we need a story for how users migrate their monitoring when this gets merged.

Reply:
Consolidating the rename discussion in the OP.